secrets.properties

```diff
@@ -0,0 +1 @@
+PLACES_API_KEY = "AIzaSyCwFNIpG2wFmKZN5r9f_ji8K3SGvjKcxOE"
```

Comment:
This should not be checked in, try and delete ASAP!

Reply:
Try to rebase and revert the commit that pushed this in, can wait until work session if unsure how!